corehq/apps/userreports/reports/specs.py

```diff
@@ -375,9 +378,46 @@ def restricted_to_static(cls):
         # so this column type is only available for static reports.  To release this,
         # we should require that conditions be expressed using a PreFilterValue type
         # syntax, as attempted in commit 02833e28b7aaf5e0a71741244841ad9910ffb1e5
-        return True
+        return False     # TODO True
 
 
+class SumWhenTemplateColumn(SumWhenColumn):
+    type = TypeProperty("sum_when_template")
+
+    @classmethod
+    def restricted_to_static(cls):
+        return False
```

Comment:
You may want to restrict this to ICDS somehow

Reply:
I feel like you don't want to say the words "feature flag"...but yeah, that's a good point, I can add one.